### ISSUE TITLE
Bluetooth only devices without a gateway?

### ISSUE THREAD

**Issue by aayushbhatia06** [0x5e/homebridge-tuya-platform#313]
### Prerequisite

- [X] Yes, I've read the readme completely.

### Cache

- [X] Yes, I've cleaned accessory cache and the issue still exists.

### Version

v1.6.0

### Device Control Mode

None

### Other Infomations

I am planning to buy the generic tuya "fingerbot plus" devices available online. 

Now, I know to use them just with Tuya, I need a bluetooth gateway, that said, I was wondering if its also true for this plugin?

In other words, can I remotely control these fingerbot devices without a bridge using this plugin?

This might sound stupid but some other plugins (like Govee plugins) allow you to remotely control bluetooth only devices by using Homebridge host pc's bluetooth

Thanks in advance!

**Comment by 0x5e** [comment]
techinially yes, as long as the sub device can be controlled by app via internet (without local network), the plugin will be able to controll it too. however there's too many products I can't promise it is available with this plugin.

**Comment by aayushbhatia06** [comment]
No, the bluetooth devices dont have internet support, to work with the tuya app either my phone needs to be in bluetooth range to the devices or I should have them linked to a tuya bluetooth hub. 
And to anyone who comes through the same issue, you CANNOT control the bluetooth devices without a hub through homekit. ALSO, most bluetooth devices I have tried are too slow to respond through homekit to be useful (My fingerbot through tuya app takes less than 1 second whereas through homekit its more like 5). So buy them at your own caution.
Still awesome work 0x5e and thank you for the very useful plugin. Closing for now

**Comment by 0x5e** [comment]
sorry I didn't read the question carefully. this plugin uses tuya cloud api, so a ble hub is required.
